flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/runtime/batch/sql/TableEnvironmentITCase.scala

```diff
@@ -231,6 +231,7 @@ class TableEnvironmentITCase(
 
     val resultFile = _tempFolder.newFile().getAbsolutePath
     result.toDataSet[(Long, Double)]
+      .map(ds => (ds._1.toString, "%.5f".format(ds._2)))
```

Comment:
nit: add a comment to avoid confusion in the future?

Reply:
I added a comment.

Side note. I know the solution is not the nicest we plan to drop the legacy planner in 1.12, that's why did not want to invest too much time in properly rewriting the test.